frontend/src/components/learner/browser/CourseCard.tsx

```diff
@@ -63,13 +77,13 @@ const CourseCard = (): React.ReactElement => {
               "-webkit-box-orient": "vertical",
             }}
           >
-            Description description description description description
-            description description
+            {hidden ? "Description..." : course.description}
           </Box>
           <Spacer />
           <Box>
             <Icon as={DocumentIcon} marginTop="-5px" marginRight="5px" />
-            insert course info
+            {hidden ? "N" : moduleCount} Modules • {hidden ? "N" : lessonCount}{" "}
```

Comment:
Would suggest to use https://github.com/uwblueprint/rowan-house/blob/main/frontend/src/components/learner/ModuleLessonCount.tsx instead